app/controllers/organizations_controller.rb

```diff
@@ -106,7 +106,7 @@ def permitted_params
   def organization_params
     params.require(:organization).permit(permitted_params)
       .transform_values do |value|
-        if value.class.name == "String"
+        if value.instance_of?("String")
```

Comment:
Was this added because of Rubocop or maybe automatically during the commit (via Husky or similar)?

I think this is causing the CI to fail. If we're going to use `.instance_of?` it should be:

```suggestion
        if value.instance_of?(String)
```

Reply:
Yea I definitely didn't manually add this. I'll adjust.